main.go

```diff
@@ -93,7 +94,7 @@ func GetCarPartsEndpoint(w http.ResponseWriter, req *http.Request) {
 
 func main() {
 	router := mux.NewRouter()
-	cluster, _ := gocb.Connect("localhost:8091")
+	cluster, _ := gocb.Connect("db:8091")
```

Comment:
maybe pass this as a flag or env var to use different hostnames?

Reply:
I have seen people use this? couchbase://localhost 

Maybe this works